src/webR/proxy.ts

```diff
@@ -49,11 +49,11 @@ type RProxify<T> = T extends Array<any>
  *   - Where an RObjImpl would be returned, an RProxy is returned instead
  *   - All return types are wrapped in a Promise
  *
- * If required, the proxy target can be accessed directly through the _target
- * property.
+ * If required, the WebRPayload object associated with the proxy can be
```

Comment:
```suggestion
 * If required, the WebRPayloadPtr object associated with the proxy can be
```

Since it doesn't make sense to wrap the other kinds of payload?